kytos/utils/config.py

```diff
@@ -52,6 +62,27 @@ def __init__(self, config_file='~/.kytosrc'):
                 os.chmod(self.config_file, 0o0600)
                 self.config.write(output_file)
 
+    @staticmethod
+    def _create_data_files_directory():
+        """Install data_files in the /etc directory."""
```

Comment:
They're not really _data_ files, but etc|config|skel ones. `_create_skel_dir` is a fine name.

Also: consider making it into a normal function instead of a `staticmethod`.

Reply:
Thanks Humberto! In this second comment, can I move the function to KytosConfig class external side or only turn this method in an instance method with self?